static/app/views/dashboardsV2/widgetBuilder/widgetBuilder.tsx

```diff
@@ -464,7 +502,7 @@ const SearchConditionsWrapper = styled('div')`
   }
 `;
 
-const StyledSearchBar = styled(SearchBar)`
+const Search = styled(SearchBar)`
```

Comment:
nit: I kinda like the pattern of prefixing the component with `Styled` so I immediately know which base component is being styled without having to scroll down the file

Reply:
I understand, but recently we talked about semantic naming at the Frontend TSC and I thought that is a good idea